opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -1947,9 +1950,9 @@ namespace {
             RG_sum = RG.sum();
         }
 
-        const double mass_balance_residual_water = fabs(BW_avg*RW_sum) * dt / pvSum;
-        const double mass_balance_residual_oil = fabs(BO_avg*RO_sum) * dt / pvSum;
-        const double mass_balance_residual_gas = fabs(BG_avg*RG_sum) * dt / pvSum;
+        const double mass_balance_residual_water = std::abs(BW_avg*RW_sum) * dt / pvSum;
+        const double mass_balance_residual_oil   = std::abs(BO_avg*RO_sum) * dt / pvSum;
+        const double mass_balance_residual_gas   = std::abs(BG_avg*RG_sum) * dt / pvSum;
```

Comment:
This is correct, but I have been using fabs() instead of abs() for the following reason: if I fail to include the proper header for std::abs, is there a risk that I get the `int abs(int)` function from C instead? Using std:: eliminates the risk of course.